core/app/models/concerns/spree/default_price.rb

```diff
@@ -5,20 +5,54 @@ module DefaultPrice
     extend ActiveSupport::Concern
 
     included do
-      has_one :default_price,
-        -> { with_discarded.currently_valid.with_default_attributes },
-        class_name: 'Spree::Price',
-        inverse_of: :variant,
-        dependent: :destroy,
-        autosave: true
+      delegate :display_price, :display_amount, :price, to: :default_price, allow_nil: true
+      delegate :price=, to: :default_price_or_build
+
+      # @see Spree::Variant::PricingOptions.default_price_attributes
+      def self.default_pricing
```

Comment:
naming nit: Should this be called `self.default_price_attributes`?

Reply:
Done!